webpack.common.js

```diff
@@ -47,12 +47,21 @@ module.exports = {
 			},
 			{
 				test: /\.(svg)$/i,
+				loader: 'url-loader',
+				exclude: '/node_modules/quill/',
+			},
+			{
+				test: /\.(svg)$/i,
+				include: '/node_modules/quill/',
```

Comment:
what's that used for?

Reply:
The svg files in from the quill module require the html-loader rather than the url-loader (they get converted to base64 when loaded with url-loader).